src/main/java/chess/service/ChessService.java

```diff
@@ -26,31 +30,46 @@ public class ChessService {
     private final GameDao gameDao;
     private final BoardDao boardDao;
 
-    @Autowired
     public ChessService(GameDao gameDao, BoardDao boardDao) {
         this.gameDao = gameDao;
         this.boardDao = boardDao;
     }
 
-    public BoardDto selectBoard() {
-        ChessBoard chessBoard = boardDao.find();
+    @Transactional
+    public int insertGame(RoomDto roomDto, ChessBoard chessBoard) {
+        ChessGame chessGame = new ChessGame(chessBoard);
+        chessGame.playGameByCommand(GameCommand.of("start"));
+
+        int id = gameDao.save(roomDto.getTitle(), roomDto.getPassword(), chessGame.getState().getValue());
+        boardDao.save(chessGame.getChessBoard(), id);
+        return id;
+    }
+
+    public List<GameDto> findGame() {
+        List<GameDto> gameDtos = new ArrayList<>();
+        List<Game> games = gameDao.findAll();
+        for (Game game : games) {
+            gameDtos.add(new GameDto(game.getId(), game.getTitle(), game.getState()));
+        }
+        return gameDtos;
+    }
```

Comment:
해당 로직을 한번 간결하게 바꿔볼 수도 있을 것 같아요!!

만약 GameDto에서 Game 객체를 받아서 GameDto를 생성하는 정적 팩토리 메서드 `of`가 있고 이를 활용해서 한번 코드를 작성해 본다면

```suggestion
    public List<GameDto> findGame() {
        return gameDao.findAll().stream()
                .map(GameDto::of)
                .collect(Collectors.toList());
    }
```

이런 식으로 간결하게 바꿔볼 수도 있을 것 같아요!! :)

> 이번 코멘트는 코드적인 부분에 대한 코멘트여서 스타일의 차이가 있을 수 있어요!! 레넌이 기존의 코드의 스타일이 좋다면 기존의 스타일을 가져가는 것도 좋아요 :) 한번 다른 관점의 코드 작성 스타일도 있다는 정도로 봐주시면 좋을 것 같아요 👍🏼